lib/src/builder/generation/parsing/declarations.dart

```diff
@@ -49,14 +49,6 @@ Iterable<BoilerplateDeclaration> getBoilerplateDeclarations(
   final states = members.states;
   final stateMixins = members.stateMixins;
 
-  // -----------------------------------------------------------------------------------------------
-  //
-  // Legacy map views
-  //
-  // Remove them so they don't get grouped with anything else and throw things off.
-  //
-  props.removeWhere((p) => p.isLegacyMapView);
```

Comment:
I can't remember why this isn't needed anymore... 😬 I might want to put it back?

Reply:
Good call, okay, I'll reinstate it and add a todo to figure out if we can remove it